viewer/config.js

```diff
@@ -530,5 +532,13 @@ Auth.initialize({
   userAutoCreateTmpl: exports.get('userAutoCreateTmpl'),
   userAuthIps: exports.get('userAuthIps'),
   s2s: true,
-  s2sRegressionTests: !!exports.get('s2sRegressionTests')
+  s2sRegressionTests: !!exports.get('s2sRegressionTests'),
+  authConfig: {
+    httpRealm: exports.get('httpRealm', 'Moloch'),
```

Comment:
Arkime instead here? And in WISE and Cont3xt?

Reply:
The value can't change because it will break all the encrypted passwords. :( Let me make sure I didn't break wise/cont3xt. :)

Reply:
So for 5.0 I think maybe we should make a goal of Moloch->Arkime most places. Maybe for this we change the default to Arkime and if you want backwards compat you need to set to Moloch manually. I'll start a 5.0 list